sdk/appconfiguration/Azure.Data.AppConfiguration/src/Generated/ConfigurationClient.cs

```diff
@@ -1501,6 +1509,13 @@ internal HttpMessage CreateGetConfigurationSettingsRequest(string key, string la
             {
                 uri.AppendQuery("snapshot", snapshot, true);
             }
+            if (tags != null && !(tags is ChangeTrackingList<string> changeTrackingList0 && changeTrackingList0.IsUndefined))
+            {
+                foreach (var param in tags)
```

Comment:
why is this called "param" and not "tag"?

Reply:
This is the generated client, and I took a brief look at autorest and I believe `param` is the name used when adding query params like this to the request.